.dockerignore

```diff
@@ -0,0 +1,4 @@
+_output
+operator
+deploy
+frontend/node_modules
```

Comment:
this will affect other Dockerfiles in this folder, is that safe @FilipB

Reply:
It should be safe as other images are build using CONTEXT ${OUTDIR}/docker, e.g. https://github.com/kiali/kiali/blob/master/make/Makefile.container.mk#L17

Reply:
I don't understand it precisely, but I believe you  :smile: